pkg/kubectl/cmd/label.go

```diff
@@ -25,12 +25,14 @@ import (
 	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 	"github.com/spf13/cobra"
 )
 
 const (
 	label_long = `Update the labels on a resource.
 
+A valid label value is consisted of letters and/or numbers with a max length of %[1]d characters.
```

Comment:
I'm probably weird in that I prefer the regex. Fine with me to stick to English, so long as the rule is concisely and accurately stated. Label must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores, up to 63 characters.